recipes/allennlp-optuna/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set version = "0.1.3" %}
+
+package:
+  name: allennlp-optuna
+  version: {{ version }}
+
+source:
+  url: https://github.com/himkt/allennlp-optuna/archive/v{{ version }}.tar.gz
+  sha256: f99501023a6f178ef1b370c65ef3b503cfbae1d960bc16b7c59da763d732ea96
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
```

How about noarch? It's fine for the staged-recipes CI to fail with missing dependencies on other platforms
```suggestion
  noarch: python
```